README.md

```diff
@@ -24,6 +24,11 @@ You can use the `docker-compose` to run the app and DB servers by executing the
 $ docker-compose up --build
 ```
 
+To run the secondary app and Servers for testing run:
```

Comment:
I'd add a bit more context here, indicating why we have this secondary instance and what happens with it, a big WARNING indicating data there is nuked as part of the usage, so it shouldn't be used for any purpose that expect data to persist for more than a few minutes.